app/src/organisms/Desktop/Camera/CameraControls/index.tsx

```diff
@@ -33,21 +61,23 @@ export function CameraControls({ onClose }: CameraControlsProps): JSX.Element {
             onClick={settings.restoreToDefault}
             buttonText={t('restore_to_default')}
           />
-          <PrimaryButton
-            onClick={() => {
-              console.log('Stubbed setting savings...')
-            }}
-          >
-            {t('save')}
-          </PrimaryButton>
+          <div className={styles.buttons}>
+            <SecondaryButton onClick={onClose}>{t('Cancel')}</SecondaryButton>
+            <PrimaryButton onClick={handleSave} disabled={isLoading}>
+              <div className={styles.save_button}>
+                {isLoading && <Icon name="ot-spinner" spin size="1rem" />}
+                {t('Save')}
```

Comment:
```suggestion
                {t('save')}
```
Nit, i18n strings should be lowercased, even if the actual translated string is capitalized